resources/i18n/en.json

```diff
@@ -173,5 +173,7 @@
     "file-not-found-text": "ZIM file doesn't exist or is not readable",
     "zim-id": "Zim Id",
     "zim-name": "Zim Name",
-    "zim-path": "Zim File Path"
+    "zim-path": "Zim File Path",
+    "export-reading-list": "Export reading list",
+    "export-reading-list-error": "An error has occured during export."
```

Comment:
@kelson42 Are you fine with such an error message? I don't like that "reading list" is missing from the error text. Also no details about the error are provided, but addressing that one will require changes to `libkiwix`, and most likely that part won't be translated.

Reply:
Changed the message as requested and waiting for confirmation.

Reply:
@veloman-yunkan Yes, message should be "An error has occured during reading list export."